test/pandas/test_pandas_tools.py

```diff
@@ -63,14 +64,100 @@ def test_write_pandas(conn_cnx: Callable[..., Generator['SnowflakeConnection', N
             cnx.execute_string("DROP TABLE IF EXISTS {}".format(table_name))
 
 
+@pytest.mark.parametrize('chunk_size', [5, 4, 3, 2, 1])
+@pytest.mark.parametrize('compression', ['gzip', 'snappy'])
+# Note: since the file will to small to chunk, this is only testing the put command's syntax
```

Comment:
Oof, I don't know what I was thinking when I wrote this...
```suggestion
# Note: since the file will be too small to chunk, this is only testing the put command's syntax
```